supportedBackends/google/pipelines/v2alpha1/src/main/resources/gcs_transfer.sh

```diff
@@ -86,64 +86,78 @@ delocalize_file_or_directory() {
 
   # required must be optional for 'file_or_directory' and was checked in the caller
   if [[ -f "$container" ]]; then
-    delocalize_file "$cloud" "$container" "$rpflag" "$required" "$content"
+    private::delocalize_file "$cloud" "$container" "$rpflag" "$required" "$content"
   elif [[ -d "$container" ]]; then
-    delocalize_directory "$cloud" "$container" "$rpflag" "$required" "$content"
+    private::delocalize_directory "$cloud" "$container" "$rpflag" "$required" "$content"
   elif [[ -e "$container" ]]; then
     echo "'file_or_directory' output '$container' exists but is neither a file nor a directory"
     exit 1
   fi
 }
 
-timestamped_message() {
+private::timestamped_message() {
   printf '%s %s\n' "$(date -u '+%Y/%m/%d %H:%M:%S')" "$1"
 }
 
-localize_message() {
+private::localize_message() {
   local cloud="$1"
   local container="$2"
   local message=$(printf "Localizing input %s -> %s" "$cloud" "$container")
-  timestamped_message "${message}"
+  private::timestamped_message "${message}"
 }
 
-delocalize_message() {
+private::delocalize_message() {
   local cloud="$1"
   local container="$2"
   local message=$(printf "Delocalizing output %s -> %s" "$container" "$cloud")
-  timestamped_message "${message}"
+  private::timestamped_message "${message}"
 }
 
+# Two transfer modes: requester pays status known and requester pays status unknown.
+#private::transfer_rp_known() {
```

Comment:
Is this intentionally left commented?

Reply:
At the time yes, but plans have changed and it might not come back so I'll remove it.